app/assets/stylesheets/modules/_page-contents.scss

```
@@ -3,7 +3,7 @@
     margin-left: govuk-spacing(0);
     padding-right: govuk-spacing(3);
 
-    &:before {
+    &:before {  // stylelint-disable-line selector-pseudo-element-colon-notation
```

I think for this we can apply the suggested change. This is not part of the issue with `media-feature-range-notation` that Martin described in the comment

```suggestion
    &::before {
```

---

But not a blocker

---

OH! Good spot, thank you. Will do that